joomla/joomla-cms
# [4.0] Removing special layouts for com_contact

The special layouts of com_contact (displaying contacts in sliders, tabs or plain) are stupid. We have no other output that you can switch like this. So this PR removes that feature.

### How to test
Create a new contact and see that you can't select a presentation style for this. Likewise in menu items and the global configuration for com_contact. See in the frontend that a contact is only displayed in plain style.
